packages/integration-testsuite/package.json

```diff
@@ -36,6 +36,7 @@
     "@josephg/resolvable": "^1.0.1",
     "body-parser": "^1.20.0",
     "express": "^4.18.1",
+    "graphql-http": "1.7.2",
```

Comment:
I think this needs some renovate config to enforce the pin due to:
https://github.com/apollographql/renovate-config-apollo-open-source/blob/master/package.json#L15

Reply:
I thiiiink this might actually be ok (after all it doesn't un-pin `@apollo/server`) but we can also find out the hard way?